Tests/Tests/SDTestCase.h

```diff
@@ -12,7 +12,7 @@
 
 #import <XCTest/XCTest.h>
 #import <Expecta/Expecta.h>
-#import <SDWebImage/SDWebImage.h>
```

Comment:
This is important. Sometimes the continue 2 test will cause header issue because of that `MapKit`. Maybe it's something wrong because of clang behavior for `__has_include`. Change this into the modular header import (which is `<SDWebImage/SDWebImage-umbrella.h>`) fix the issue.